src/cal/test_atmos_corr_raw_to_rad.py

```diff
@@ -325,21 +347,25 @@ def main():
     Return:
     None
     """
+
+    # Prints the directory the script is in
+    print("Current directory: " + os.path.dirname(os.path.abspath(__file__)))
     
     # Finds the current directory and appends a new folder to be made
-    current_dir = os.path.dirname(os.path.abspath(__file__))
-    final_dir = os.path.join(current_dir, 'Output Files')
-    # If the folder doesn't exist, then it will be made.
-    if not os.path.exists(final_dir):
-        os.makedirs(final_dir)
+    working_dir = input("Please input the FULL path to the directory with the images: \n")

```

Comment:
Instead of asking for the user to provide input as the script runs, you can add input arguments. An example can be found [here](https://github.com/iceberg-project/Seals/blob/master/src/entk_script/entk_script.py#L154). 

Eventually this script will run without any interaction by the user. As a result any input during execution time will not be possible. Providing everything during the python call is preferable. It will also minimize the changes that may be needed for when we run it on a Supercomputer